tests/buffer/global_buffer_set_final_data.cpp

```diff
@@ -9,38 +9,35 @@
 using namespace cl::sycl;
 
 
-constexpr size_t N = 16;
 
-buffer<int> b { N };
 
-int test_main(int argc, char *argv[]) {
+int test_main(int /*argc*/, char** /**argv[]*/) {
+  constexpr size_t N = 16;
+  
+  buffer<int> buf{ N };
```

Comment:
As you can see in the test name and in the comments you removed, this is for testing a global buffer case.

If you make the buffer an automatic variable, well, it is another test...